html/partials/500.shtml

```diff
@@ -1,2 +1,10 @@
-<h2>500 Internal server error</h2>
-<p>There's a problem. Please try again in a few moments.</p>
+<p><strong><h1></h1>Something's gone wrong with our servers</h1></strong></p>
+<p>We're fixing this as quickly as possible.</p>
+<p>Please try again.</p>
+<p>If you're still having difficulty:</p>
+<!--#if expr="$display_contact_details = TRUE" -->
+<ul>
+	<li>phone the IT service desk on 0845 00 00 050 or;</li>
+	<li>email ITServiceDesk@homeoffice.gsi.gov.uk</li>
```

Comment:
These details will be specific to your project. It is not acceptable to these to be hard-coded into this repo as it is used my multiple projects.

Instead we should allow users to set their own contact details via environment variables.